### ISSUE TITLE
gulp-sri-hash (unnecessarily) escapes unicode characters

### ISSUE THREAD

**Issue by amyspark** [macedigital/gulp-sri-hash#23]
With this plugin,
```html
<title>Main Page | Amyspark’s Domain</title>
```
becomes
```html
<title>Main Page | Amyspark&#x2019;s Domain</title>
```

I think this is because `cheerio` is decoding (and reencoding) non-ASCII entities by default. Can this be fixed?

**Comment by aaron-russell** [comment]
I created a pull request to hopefully fix this #26

**Comment by macedigital** [comment]
A new release v2.1.0 is available from your preferred npm-mirror, which sets `{ decodeEntities: false }` option to cheerio. @aaron-russell Thanks again for your PR.
